test/testrunner.py

```diff
@@ -449,8 +449,13 @@ def main():
             sys.exit(pytest.main(pytest_cmd))
 
         else:
+            sm_remote_images = []
+            for std_img in standard_images_list:
+                # Do not run SM tests on DIY images
+                if "diy" not in std_img:
+                    sm_remote_images.append(std_img)
             run_sagemaker_remote_tests(
-                [image for image in standard_images_list if not ("tensorflow-inference" in image and "py2" in image)]
+                [image for image in sm_remote_images if not ("tensorflow-inference" in image and "py2" in image)]
```

Comment:
What if we add this filtering condition into if condition [here](https://github.com/aws/deep-learning-containers/pull/1278/files#diff-7cbc819cfee09f8be65c72606589778a1d44839986c87dfb0930a40ab75820efR455)?

Reply:
sure, added